libavcodec/cbs_h266.h

```diff
@@ -848,6 +848,27 @@ typedef struct H266RawSlice {
     int          data_bit_start;
 } H266RawSlice;
 
+typedef struct H266RawSEIFilmGrainCharacteristics {
```

Comment:
could we reuse AVFilmGrainH274Params?

Reply:
better not to use here. It's used to align for h264/h265/h266 syntax_template.